### ISSUE TITLE
Bug on styles building with webpack

### ISSUE THREAD

**Issue by arthurkushman** [facebook/draft-js#1498]
**Do you want to request a *feature* or report a *bug*?**
Bug
**What is the current behavior?**
ERROR in ./~/css-loader?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!./~/style-loader!./~/css-loader!./~/draft-js-inline-toolbar-plugin/lib/plugin.css
**What is the expected behavior?**
Styles should be loaded and toolbar appeared adjusted to the block.
**Which versions of Draft.js, and which browser / OS are affected by this issue? Did this work in previous versions of Draft.js?**
Draft version 0.10.4, OS Ubuntu, Browser Chrome 59.0.3071.109 (Official Build) (64-bit)

```js
ERROR in ./~/css-loader?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!./~/style-loader!./~/css-loader!./~/draft-js-inline-toolbar-plugin/lib/plugin.css
Module build failed: Unknown word (5:1)

  3 | // load the styles
  4 | var content = require("!!./../../css-loader/index.js!./plugin.css");
> 5 | if(typeof content === 'string') content = [[module.id, content, '']];
    | ^
  6 | // add the styles to the DOM
  7 | var update = require("!./../../style-loader/addStyles.js")(content, {});
  8 | if(content.locals) module.exports = content.locals;

 @ ./~/draft-js-inline-toolbar-plugin/lib/plugin.css 4:14-198 13:2-17:4 14:20-204
 @ ./src/components/form/fields/CustomToolBarEditor.js
 @ ./src/components/form/fields/Draft.js
 @ ./src/components/form/Editor.js
 @ ./src/Reactables.jsx
 @ ./main.js
 @ multi (webpack)-dev-server/client?http://localhost:8888 webpack/hot/dev-server ./main.js

ERROR in ./~/css-loader?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!./~/style-loader!./~/css-loader!./~/draft-js-image-plugin/lib/plugin.css
Module build failed: Unknown word (5:1)

  3 | // load the styles
  4 | var content = require("!!./../../css-loader/index.js!./plugin.css");
> 5 | if(typeof content === 'string') content = [[module.id, content, '']];
    | ^
  6 | // add the styles to the DOM
  7 | var update = require("!./../../style-loader/addStyles.js")(content, {});
  8 | if(content.locals) module.exports = content.locals;

 @ ./~/draft-js-image-plugin/lib/plugin.css 4:14-198 13:2-17:4 14:20-204
 @ ./src/components/form/fields/CustomToolBarEditor.js
 @ ./src/components/form/fields/Draft.js
 @ ./src/components/form/Editor.js
 @ ./src/Reactables.jsx
 @ ./main.js
 @ multi (webpack)-dev-server/client?http://localhost:8888 webpack/hot/dev-server ./main.js
```

**Comment by thibaudcolas** [comment]
Hey @arthurkushman, it's very hard to follow your code sample but it looks like you're submitting this on the wrong project. Your stack traces point at `draft-js-inline-toolbar-plugin` and `draft-js-image-plugin`, so you're after https://github.com/draft-js-plugins/draft-js-plugins not Draft.js itself.

**Comment by arthurkushman** [comment]
Solved all problems by switching to rte, thx
